torch/ao/quantization/pt2e/lowering.py

```diff
@@ -0,0 +1,60 @@
+import torch
+from torch._inductor.constant_folding import constant_fold
+from torch._inductor.fx_passes.freezing_patterns import freezing_passes
+
+
+__all__ = [
+    "lower_pt2e_quantized_to_x86",
+]
+
+
+def lower_pt2e_quantized_to_x86(
```

Comment:
Similar to other API around here. Why is this a special x86 API? Can't we integrate this in the default lowering passes so that users benefit from it by default? It is unlikely they will discover this function otherwise :/

Reply:
Thanks for your comments. This API is for x86 only. And there isn't a default lowering pass for PT2E quantization. The PT2E quantization workflow does not provide any lowering pass. Lowering passes are provided by backends. @jerryzh168 may comment if needed. Thanks.

Reply:
BTW, do you know how to fix the doc issue? Thanks.

Reply:
@albanD this is kind of a semi private API to enable some internal use cases, so it's fine for this not to be very discoverable I think